example/config/README.md

```diff
@@ -0,0 +1,249 @@
+            - `value`
+        - `max`: specifying `max` here will take precedence over the `all` rule for aggregations using a MAX() function
+    - `aggregates`: aggregates and categoricals define the actual features created. So at least one is required. Aggregates of numerical columns.
+        - (First quantity)
+            - `quantity`: Each quantity is a number of some and the list of metrics are applied to each quantity.
+            - `imputation`:  Imputation rules specified at the level of specific features will take precedence over the higer-level rules specified above. Note that the 'count' and 'sum' metrics will be imputed differently here.
+                - `count`:
+                    - `type`: `mean`
+                - `sum`:
+                    - `type`: `constant`
+                    - `value`: `137`
+            - `metrics`:
+                - `count`
+                - `sum`
+            - `coltype`: `smallint` (Optional, if you want to control the column type in the generated features tables)
+        - (Second quantity)
+            - `quantity`: `some_flag` (Since we're specifying `aggregates_imputation` above, a feature-specific imputation rule can be omitted)
+            - `metrics`: 
+                - `max`
+                - `sum`
+
+    - `categoricals`:  Categorical features. The column given can be of any type, but the choices must comparable to that type for equality within SQL The result will be one feature for each choice/metric combination.
+    - (First column)
+        - `column`: Note that we haven't specified a top level `categoricals_imputation` set of rules, so we have to include feature-specific imputation rules for both of our categoricals here.
+        - `imputation`:
+            - `sum`:
+                - `type`: `null_category`
+            - `max`:
+                - `type`: `mean`
+        - `choices`:
+        - `metrics`: `sum`
+    - (Second column)
+        - `column`: `shape` (As with the top-level imputation rules, `all` can be used for the feature-level rules to specify the same type of imputation for all aggregation functions)
+        - `imputation`:
+            - `all`:
+                `type`: `zero`
+        - `choice_query`: `select distinct shape from cool stuff`
+        - `metrics`: 
+            - `sum`
+    - `intervals`: The time intervals over which to aggregate features
+    - `groups`: A list of different columns to separately group by
+
+### Feature Grouping
+define how to group features and generate combinations
+
+- `feature_group_definition`: feature_group_definition allows you to create groups/subset of your features by different criteria. for instance,
+    - `tables`: allows you to send a list of collate feature tables (collate builds these by appending `aggregation_imputed` to the prefix)
+    - `prefix`: allows you to specify a list of feature name prefixes
```

Comment:
Definitely note that triage will consider prefixes sharing their initial string  as the same (e.g.,`bookings` will encompass `bookings_charges` as well as `bookings`). In other words, it's not actually maintaining a list of prefixes and searching for exact matches but trusting that prefixes do not overlap in this way.